src/Tools/BuildBoss/InternalsVisibleTo.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Xml.Linq;
+
+namespace BuildBoss
+{
+    internal readonly struct InternalsVisibleTo
+    {
+        public InternalsVisibleTo(string targetAssembly, string publicKey, string loadsWithinVisualStudio)
+        {
+            TargetAssembly = targetAssembly;
+            PublicKey = publicKey;
+            LoadsWithinVisualStudio = loadsWithinVisualStudio;
+        }
+
+        public string TargetAssembly { get; }
+        public string PublicKey { get; }
+        public string LoadsWithinVisualStudio { get; }
```

Comment:
Consider just making this a Boolean so that way the logic for parsing it is in one place versus potentially multiple consumption points.

Reply:
I tried this but the result didn't seem to be a significant improvement over the current form.